streams/src/commonMain/kotlin/com/mirego/trikot/streams/reactive/ColdPublisher.kt

```diff
@@ -27,8 +27,6 @@ class ColdPublisher<T>(private val executionBlock: ColdPublisherExecutionBlock<T
     override fun onNoSubscription() {
         super.onNoSubscription()
         cancellableManagerProvider.cancelPreviousAndCreate()
-        if (!completed) {
-            cleanupValues()
-        }
+        cleanupValues()
```

Comment:
If the parent publisher is completed, the value must be kept. This is a major breaking change and I don't understand why a parent publisher state would change if its state is completed ?